src/github.com/matrix-org/go-neb/services/github/github.go

```diff
@@ -142,55 +141,54 @@ func (s *Service) cmdGithubCreate(roomID, userID string, args []string) (interfa
 	return gomatrix.TextMessage{"m.notice", fmt.Sprintf("Created issue: %s", *issue.HTMLURL)}, nil
 }
 
-func (s *Service) cmdGithubComment(roomID, userID string, args []string) (interface{}, error) {
+const cmdGithubReactUsage = `!github react [owner/repo]#issue (+1|-1|laugh|confused|heart|hooray)`
```

Comment:
as in someone posts an issue and we just do `:+1:`
that'd require rewriting most of the service to be aware of the most recent issue created

I'm just using the strings that the API lists as valid